frontend/css/brainzplayer.less

```diff
@@ -351,6 +351,23 @@
   }
 }
 
+.volume {
```

Comment:
One more style suggestion to be applied to the whole website, so to be added to the frontend/css/main.less file:

```less
input[type='range'] {
    accent-color: @blue;
}
```

This will style the color of the range input:
![image](https://github.com/user-attachments/assets/c09c6c0c-aa2c-401c-91ec-21722d55b189)